CONTRIBUTING.md

```diff
@@ -206,28 +197,29 @@ There may be times that a test is already written - in these circumstances it is
 
 ## Merge Requests
 
-We welcome merge requests with fixes and improvements to the project.  The features we really would like public support on are marked with "up-for-grabs" but other improvements are also welcome - please ensure you read over the merge work-flow below.
+We welcome merge requests with fixes and improvements to the project.  The features we really would like public support on are marked with ["up-for-grabs"](https://vatsimuk.atlassian.net/browse/CORE-26?jql=project%20%3D%20CORE%20AND%20labels%20%3D%20up-for-grabs%20order%20by%20lastViewed%20DESC) but other improvements are also welcome - please ensure you read over the merge work-flow below.
+
+If you wish to add a new feature or you spot a bug that you wish to fix, **please open an issue for it first** in the [CORE project](https://vatsimuk.atlassian.net/projects/CORE/issues).
 
-If you wish to add a new feature or you spot a bug that you wish to fix, **please open an issue for it first** on the [UK Core Web Services Repository](https://github.com/vatsim-uk/core/issues).
+**Note:** You will need to signup to create issues within the project. See [Issue Tracking](ISSUE_TRACKING.md) for more information.
 
 The work-flow for submitting a new merge request is designed to be simple, but also ensure consistency from **all** contributors:
 
 * Fork the project into your personal space on GitHub.com
 * Create a new branch (with the name `<issue_number>-<name>`, replacing issue_number with the issue number you're resolving)
  * The exception to this is where an entire feature is being tackled, spanning multiple issues.  In this case you can create a `feature/<name>` branch.
```

Comment:
If working on a feature branch, will these also be linked to an issue if we call them `feature/core-##-my-great-feature`?

Reply:
Yes. Although I would reckon that in the case of a feature, the Epic issue should be linked and not the individual issues within it.

Reply:
Agree completely 👍